src/administrative-unit/administrative-unit.controller.ts

```diff
@@ -14,8 +14,18 @@ export class AdministrativeUnitController {
     return await this.adminService.getDistricts(id);
   }
 
+  @Get('all-districts')
+  async findAllDistricts() {
+    return await this.adminService.getAllDistricts();
+  }
+
   @Get('wards/:id')
   async findWard(@Param('id', ParseIntPipe) id: number) {
     return await this.adminService.getWards(id);
   }
+
+  @Get('all-wards')
```

Comment:
```suggestion
  @Get('wards')
```

xem lại cách viết RESTAPI